legend-engine-test-runner-service/src/main/java/org/finos/legend/engine/testable/service/assertion/RelationalTestAssertionHelper.java

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.testable.service.assertion;
+
+import org.finos.legend.engine.plan.execution.result.serialization.SerializationFormat;
+import org.finos.legend.engine.plan.execution.stores.relational.result.RelationalResult;
+
+public class RelationalTestAssertionHelper
+{
+    RelationalResult relationalResult;
+
+    public RelationalTestAssertionHelper(RelationalResult relationalResult)
+    {
+        this.relationalResult = relationalResult;
+    }
+
+    public String flushToJSONString()
+    {
+        try
+        {
+            return relationalResult.flush(relationalResult.getSerializer(SerializationFormat.PURE_TDSOBJECT));
+        }
+        catch (RuntimeException error)
+        {
```

Comment:
why do we use pure_tds object as default serialization. Seems a bit off from naming, should we sue default?

Reply:
How about we use the assert type for this. 
If it is `AssertToJSON` we use the default
Moving to assertToTDS we use the `TDS` serialization for this. 
what are your thoughts for doing that. 
I think it might be an overkill to model another assert for TDS.

Reply:
@hardikmaheshwari Rethinking it, I dont think we should expose this to test. I think the asserts should take care of this. 
If use is using EqualToTDS then we can use the TDS serialization. For now we will support EqualToTDS with json input. We can further improve this by having user type out CSV but for now I think it is fairly clear for the user this way. 
Thoughts?